models/topics.model.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+
+exports.fetchTopics = async () => {
+  let queryStr = `SELECT * FROM topics`;
+  let queryVals = [];
```

Comment:
Declare arrays with const.

Is this needed in this query? All we want is all topics, I can't see any values that need to be parametrised?